lib/gradient.coffee

```diff
@@ -103,10 +103,11 @@ class PDFGradient
         Type: 'XObject'
         Subtype: 'Form'
         FormType: 1
-        BBox: [0, 0, @doc.page.width, @doc.page.height]
+        BBox: [-1000000, -1000000, 1000000, 1000000]
```

Comment:
I agree it is more an ugly hack than a real fix. And it doesn't solve an other issue with gradients: the opacity of the gradient is lost if the shape is filled and stroked...
Using a "Tiling Pattern" with the dimensions of the page could be a better solution (they are unaffected by the page transformations so there wouldn't be the clipping problem). I close this P.R. and will open a new one if I have something better